apps/testingaccessibility/src/pages/thanks/purchase.tsx

```diff
@@ -102,7 +102,9 @@ const ThanksVerify: React.FC<React.PropsWithChildren<ThanksProps>> = ({
   bulkCouponId,
 }) => {
   const isTeamPurchase =
-    purchaseType === NEW_BULK_COUPON || purchaseType === EXISTING_BULK_COUPON
+    purchaseType === NEW_BULK_COUPON ||
+    purchaseType === EXISTING_BULK_COUPON ||
+    purchaseType === INDIVIDUAL_TO_BULK_UPGRADE
   const isNewPurchase =
     purchaseType === NEW_BULK_COUPON || purchaseType === NEW_INDIVIDUAL_PURCHASE
```

Comment:
Here is the change that gets the post-purchase page displaying the right thing when a user is upgrading from individual to bulk/team.